egs2/chime8_task1/README.md

```diff
@@ -52,21 +52,24 @@ significantly complicates speaker counting.
 
 
 ```bash
+###############################################################################
+### tcpWER for all Scenario ###################################################
+###############################################################################
 +-----+--------------+--------------+----------+----------+--------------+-------------+-----------------+------------------+------------------+------------------+
 |     | session_id   |   error_rate |   errors |   length |   insertions |   deletions |   substitutions |   missed_speaker |   falarm_speaker |   scored_speaker |
 |-----+--------------+--------------+----------+----------+--------------+-------------+-----------------+------------------+------------------+------------------|
-| dev | chime6       |     0.825381 |    52070 |    63086 |        12747 |       29466 |            9857 |                0 |                5 |                8 |
-| dev | mixer6       |     0.287729 |    26621 |    92521 |         4882 |        8809 |           12930 |                0 |               24 |               70 |
-| dev | dipco        |     0.674161 |    11574 |    17168 |         3066 |        5563 |            2945 |                0 |                2 |                8 |
-| dev | notsofar1    |     0.508768 |    90660 |   178195 |        14872 |       55195 |           20593 |              105 |                7 |              592 |
+| dev | chime6       |     0.887079 |    54417 |    61344 |        14542 |       29683 |           10192 |                0 |                4 |                8 |
+| dev | mixer6       |     0.292346 |    26317 |    90020 |         4656 |        9079 |           12582 |                0 |               23 |               70 |
+| dev | dipco        |     0.984403 |    16157 |    16413 |         5008 |        8507 |            2642 |                3 |                0 |                8 |
+| dev | notsofar1    |     0.462402 |    70035 |   151459 |        13116 |       37324 |           19595 |              120 |                5 |              612 |
 +-----+--------------+--------------+----------+----------+--------------+-------------+-----------------+------------------+------------------+------------------+
 ###############################################################################
 ### Macro-Averaged tcpWER for across all Scenario (Ranking Metric) ############
 ###############################################################################
 +-----+--------------+
 |     |   error_rate |
 |-----+--------------|
-| dev |      0.57401 |
+| dev |     0.656558 |
```

Comment:
I think this number has too many decimal points?

Reply:
must be an issue with tabulate. 
I can fix it

Reply:
I think it is better to leave as it is sorry. 
Because I ve to fix it via `https://github.com/chimechallenge/chime-utils` and then if people do not re-install it they will get slighly different values and be puzzled why they do not correspond. 
It is the output of the scoring script.

Reply:
I know it looks a bit odd

Reply:
ok, that makes sense.